### ISSUE TITLE
sql: statements detail page isn't working 

### ISSUE THREAD

**Issue by awoods187** [cockroachdb/cockroach#56252]
When I click on the sessions -> statements detail page for a given session, it won't load:
<img width="1499" alt="Screen Shot 2020-11-03 at 12 24 59 PM" src="https://user-images.githubusercontent.com/22278911/98019332-9c246a00-1dcf-11eb-85b0-1314b1a514cb.png">

Video: https://drive.google.com/file/d/1SeVhS9lvoo79jnQzvTUH2p6GpngiXJGO/view?usp=sharing
Version: V20.2.0-ALPHA.1-4562-GECA8746C4A-DIRTY

Repro:
```SQL
cd go/src/github.com/cockroachdb/cockroach
roachprod create local
roachprod put local cockroach 
roachprod put local ./bin/workload
roachprod start local
http://localhost:26264/#/overview/list
roachprod run local:4 -- "./cockroach workload fixtures import tpcc --warehouses=10 --db=tpcc"
roachprod run local:4 "./workload run tpcc --wait false --tolerate-errors --warehouses=10 --active-warehouses=10 --duration=15m --split --scatter {pgurl:1-3}"
```

**Comment by jordanlewis** [comment]
Looks like this was introduced by #55959 

@asubiotto, it's happening because of this code:

```
              ].filter(function (r) {
                if (r.name === "Network Bytes Sent" && r.value.mean === 0) {
                  // Omit if empty.
                  return false;
                }
                return r.value;
              })}
```

In this case, looks like `r.value` was null, which I think means that `stats.bytes_sent_over_network` was null. Is that expected? If not, we should look into why - and if it is expected we'll need some logic to turn null into `n/a` or something.

**Comment by asubiotto** [comment]
The new code made a pre-existing bug a lot more obvious. The bug seems to be that `r.value` is undefined even in commits prior to #55969. This only seems to happen when running tpcc, since navigating to the execution stats in the statements page after a single query on cockroach demo seems unaffected. This is what the issue looks like:

![image](https://user-images.githubusercontent.com/10560359/98552751-21f85780-229f-11eb-9bc4-45a5dabdff7d.png)

Putting up a PR to close this issue and creating a new one to track this other issue.

**Comment by asubiotto** [comment]
Closed by #56445
